pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator_test.go

```diff
@@ -0,0 +1,34 @@
+package mutators
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestCertAppanding(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	providedCa, err := ioutil.ReadFile("originalCert.pem")
+	if err != nil {
+		panic(err)
+	}
+	newCa, err := ioutil.ReadFile("newCert.pem")
```

Comment:
the names 'providedCa' and 'newCa' is a bit confusing - maybe we should write here, even just for making things more clear, that we don't expect the provided Ca to contain the 'newCa' in practice? (because users are not supposed to provide both certificates) or change the 'providedCa' to include a single certificate?

Reply:
good point, since the tests are only testing each part separately, we need to have the file with both of the certs to check that the contains function is working correctly for the flow that engineCa is parsed differently (diffrent lines count). In that case how about changing the names of the files to something like "completeCerts" and "engineerCert"?

Reply:
changing the names to better reflect their purpose sound good but did you mean the second is 'engineCert'?